src/go/rpk/pkg/cli/cmd/debug/bundle/bundle.go

```diff
@@ -158,9 +161,15 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 	command.Flags().DurationVar(
 		&timeout,
 		"timeout",
-		10*time.Second,
+		15*time.Second,
 		"How long to wait for child commands to execute (e.g. '30s', '1.5m')",
 	)
+	command.Flags().DurationVar(
+		&metricInterval,
+		"metrics-interval",
+		15*time.Second,
```

Comment:
wdyt about making this default 10 ?

Reply:
@travisdowns is 10 enough or should we push it to 15?

Reply:
I don't think there's much difference between 10 and 15. Maybe 10 is slightly better since I can do mental math a lot quicker when dividing by 10.